### DISCUSSION TITLE
Walrus Model - 去中心化的3D模型存储仓库

### DISCUSSION THREAD

**Discussion by MuyeC** [creatorsand/.github#118] [Ideas]
**项目描述：**

一个去中心化的3d模型存储工具

**代码仓库：**

https://github.com/MuyeC/walrus-model

**成员列表：**

Ra1nB0 ：合约&前端开发

**社群昵称：**

微信：Ra1nB0

**个人简介：**

半年前端。

**产品是给谁用的？**

3D艺术创作者：需要安全存储、加密和备份的个人。
有模型需求的开发者：提供便捷下载模型的渠道。

**其他产品有什么痛点，用户为什么要选择你的产品？**

1.解决创作者的存储需求，去中心化的存储，比网盘更加安全可靠。
2.提供无平台的分享服务，点对点快捷分享，免去例如Sketchfab，Blendswap等中介平台。

**Comment by v1xingyue** [comment]
ai打分

[a comment by an automated account is not shown]
